src/main/java/org/skriptlang/reflect/syntax/event/elements/StructCustomEvent.java

```diff
@@ -0,0 +1,186 @@
+package org.skriptlang.reflect.syntax.event.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEventInfo;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.lang.util.SimpleEvent;
+import ch.njol.skript.log.SkriptLogger;
+import ch.njol.skript.registrations.Classes;
+import org.skriptlang.reflect.syntax.CustomSyntaxStructure;
+import com.btk5h.skriptmirror.skript.custom.SyntaxParseEvent;
+import com.btk5h.skriptmirror.util.SkriptReflection;
+import com.btk5h.skriptmirror.util.SkriptUtil;
+import org.skriptlang.reflect.syntax.event.BukkitCustomEvent;
+import org.skriptlang.reflect.syntax.event.EventSyntaxInfo;
+import org.skriptlang.reflect.syntax.event.EventTriggerEvent;
+import org.skriptlang.skript.lang.entry.EntryContainer;
+import org.skriptlang.skript.lang.script.Script;
+
+import java.util.*;
+
+public class StructCustomEvent extends CustomSyntaxStructure<EventSyntaxInfo> {
+
+  public static boolean customEventsUsed = false;
+
+  static {
+    String[] syntax = {
+      "[(1¦local)] [custom] event %string%"
+    };
+    Skript.registerStructure(StructCustomEvent.class, customSyntaxValidator()
+        .addEntry("pattern", null, true)
+        .addEntry("event-values", null, true)
```

Comment:
What do you think about supporting `event values`? Would require some additional work to keep backwards compatibility but this is a weird design choice.

Reply:
I will look into it

Reply:
maybe this is another thing that could be an issue for the future